packages/extension/src/contexts/index.ts

```diff
@@ -2,4 +2,5 @@ export * from './BrowserContext';
 export * from './LedgerContext';
 export * from './NetworkContext';
 export * from './ServerContext';
+export * from './TransactionProgress';
```

Comment:
Could you please rename it to: TransactionProgressContext please?

Reply:
Done